### DISCUSSION TITLE
Will it gonna work on laravel 11?

### DISCUSSION THREAD

**Discussion by sameedkun** [zfhassaan/payfast#7] [Q&A] [answered]
can this work on laravel 11 and php 8.2+

**Comment by zfhassaan** [comment]
Yes.

**Comment by netbee1136** [comment]
Even its not working on 10 Larvel having classes issues

**Reply by zfhassaan** [reply]
What issues you're facing?

**Comment by zfhassaan** [comment, marked as answer]
@netbee1136 version is updated and compatibility tests are also added for laravel 9, 10, and 11. Update the package and then review.
